qemu/tests/qmp_command.py

```diff
@@ -262,10 +245,6 @@ def send_cmd(cmd):
     cmd_return_value = params.get("cmd_return_value")
     exception_list = params.get("exception_list", "")
 
-    # HOOKs
-    if result_check == 'qmp_cpu':
-        pre_cmd = "cpu index=%d" % (int(params['smp']) - 1)
-
```

Comment:
Please explain the reason for deleting "pre_cmd " and the "Current" check.  You can add it in the description for this patch, thanks !

If remove "Current" checking, no need to leave the function " qmp_cpu_check " i think, you can use the existing function "check_result" and add a few change for cpu check, such as:

if result_check == 'qmp_cpu':
    cmd_return_value = params['smp']
    result_check == "equal"
    check_result(len(output), cmd_return_value, exception_list)